src/openepd/model/specs/generated/finishes.py

```diff
@@ -495,6 +497,10 @@ class PlasterV1(BaseOpenEpdHierarchicalSpec):
 
     _EXT_VERSION = "1.0"
 
+    # Own fields:
+    composition: PlasterComposition | None = pyd.Field(default=None, description="", example="Cement")
```

Comment:
Is this correct that description is missing?

Reply:
Other enum properties use string as example, but it will be better if @OlehChyhyryn verify this

Reply:
See other comment below; I think strings are here for a reason.

Reply:
They are more error-prone to refactoring - but not for a long shot. As soon as this thing reaches the openapi vlaidator, it will complain that the example is not in allowed values.